controller/tests/models/domain/rsptag_test.py

```diff
@@ -168,34 +172,41 @@ def test_from_str() -> None:
             "display_name": "Release r21.0.1",
             "version": VersionInfo(21, 0, 1),
             "cycle": None,
+            "date": None,
         },
         "r22_0_0_rc1": {
             "tag": "r22_0_0_rc1",
             "image_type": RSPImageType.CANDIDATE,
             "display_name": "Release Candidate r22.0.0-rc1",
             "version": VersionInfo(22, 0, 0, "rc1"),
             "cycle": None,
+            "date": None,
         },
+        # fromisocalendar() gives you a naive object, even if you
+        # replace the tzinfo.
         "w_2021_22": {
             "tag": "w_2021_22",
             "image_type": RSPImageType.WEEKLY,
             "display_name": "Weekly 2021_22",
             "version": VersionInfo(2021, 22, 0),
             "cycle": None,
+            "date": datetime.datetime(2021, 6, 3, 0, 0, tzinfo=datetime.UTC),
```

Comment:
Here and elsewhere, you omit the hour and minute when creating the date, so I would omit them here as well when comparing the date, since you're relying on the default values of zero. (And you're omitting seconds anyway so may as well omit hours and minutes.)